migration/idempotent/009-tracing-views.sql

```diff
@@ -13,7 +12,7 @@ SELECT
     s.end_time,
     tstzrange(s.start_time, s.end_time, '[]') as time_range,
     s.duration_ms,
-    s.span_tags,
+    _ps_trace.tag_map_denormalize(s.span_tags) as span_tags,
```

Comment:
In the previous implementation, this column contained the tag_map rather than the actual tag_keys->tag_values. It was left to the user to decide whether to denormalize the tags. Have we verified that postgres will omit the call to tag_map_denormalize if this column is not selected?

Reply:
we are denormalizing the tags in this view, but not in the event or link views. Is there a reason to do it in one and not the others?

Reply:
> In the previous implementation, this column contained the tag_map rather than the actual tag_keys->tag_values. It was left to the user to decide whether to denormalize the tags. Have we verified that postgres will omit the call to tag_map_denormalize if this column is not selected?

No, we didn't


> we are denormalizing the tags in this view, but not in the event or link views. Is there a reason to do it in one and not the others?

We probably didn't discuss what views require update hence others fell off the radar. Will add them in a follow-up PR.